### ISSUE TITLE
Move more pages to the site?

### ISSUE THREAD

**Issue by L1lith** [melonjs/melonjs.org#10]
I'm posting this before my PR for Astro is accepted, but assuming it's accepted it might be a good things to move as much as possible to the new site (potentially including the docs and the gallery?) This would be easier now that we have Astro integration as we can do better code sharing and code splitting via modules

**Comment by L1lith** [comment]
The wiki is now partially migrated, and builds without failing. However I think a good amount of the styles are currently messed up, and there needs to be a nav bar made probably.

**Comment by L1lith** [comment]
Maybe we can find a way to setup the docs to build into the main site. We could probably take the static output from the main melonJS docs functionality and copy it in the `public` folder of the site. We just need to figure out how to set them up together. It would be nice to have everything under one domain with the same nav bar.
